### DISCUSSION TITLE
Will there be translated documentation?

### DISCUSSION THREAD

**Discussion by ghost** [maximmax42/Discord-CustomRP#123] [General]
Since the app is translated in different languages, will the app documentation also be translated in those languages?

**Comment by maximmax42** [comment]
I need to figure out how to do it first. Adding it as POEditor terms is not an option really, don't think I could add huge walls of text there. I could spin up my own wiki site, but for just a couple of pages, is it really reasonable? Oh and there's no way that I know of to make GitHub wiki open for pull requests.
